### PR TITLE
fix build with Ogre2

### PR DESCRIPTION
**Pull request by rpaciorek**
* add FindOGRE to cmake build (Ogre2 don't install cmake config file)
* use macro OGRE_LOG_ERROR instead of direct use logError (no logError in Ogre2)
* re-add `Ogre::SceneManager*` to `OgreOggListener` (in Ogre2-only part) -  constructor of MovableObject in Ogre2 need SceneManager

### CODE REVIEW

**Comment by paroj** [comment]
> add FindOGRE to cmake build (Ogre2 don't install cmake config file)

this is a Ogre-Next bug, that should not be fixed here. Please drop this.

File: oggsound/src/OgreOggISound.cpp [hunk 1]
```diff
@@ -723,8 +723,11 @@ namespace OgreOggSound
 			#endif
 		);
 
+		#if OGRE_VERSION_MAJOR == 1
```
Comment (paroj) [inline oggsound/src/OgreOggISound.cpp:726]:
```suggestion
		#if OGRE_VERSION_MAJOR != 2
```
Ogre 13 is OGRE_VERSION_MAJOR=13

**Comment by rpaciorek** [comment]
> 



> > add FindOGRE to cmake build (Ogre2 don't install cmake config file)
> 
> this is a Ogre-Next bug, that should not be fixed here. Please drop this.

Maybe bug, maybe not bug – providing specific config file for external build system is nice, but not obligatory.

If you don't want `FindOGRE.cmake` in this project repo (IMHO convenient for users), maybe use `find_package(OGRE OPTIONAL_COMPONENTS Bites)` instead of `find_package(OGRE OPTIONAL_COMPONENTS Bites CONFIG)` then it's possible use `FindOGRE.cmake` provided by Ogre-next by `-DCMAKE_MODULE_PATH=/usr/local/lib/OGRE/cmake/`

File: oggsound/src/OgreOggISound.cpp (hunk 1 above)
Comment (rpaciorek) [reply oggsound/src/OgreOggISound.cpp]: fixed

**Pull request merged**